templates/nextflow-aurora-mysql.yaml

```diff
@@ -103,6 +106,15 @@ Resources:
         PasswordLength: 16
       KmsKeyId: !Sub ${AuroraKeyStack.Outputs.Key}
 
+  AuroraMasterSecretRotationSchedule:
+    Type: AWS::SecretsManager::RotationSchedule
+    Properties:
+      SecretId: !Ref AuroraMasterSecret
```

Comment:
I was getting a linting warning that this property depends on a resource with a condition (`Condition: UseSnapshotFalse`), so there's a risk that the referred resource doesn't exist. You might want to add `Condition: UseSnapshotFalse` to this resource as well.

Reply:
done.